### PR TITLE
Update mkdocs.yml

### PR DESCRIPTION
**Pull request by rodorodo5**
Added GA Code

### CODE REVIEW

**Review by cesa2149 — APPROVED** [review]

**Review by cesa2149 — COMMENTED** [review]
approved

**Review by rodorodo5 — COMMENTED** [review]
+1

**Review by rodorodo5 — COMMENTED** [review]
+1

**Review by cesa2149 — APPROVED** [review]
apporved

**Comment by cloudnull** [comment]
Once the spacing is fixed in the file this looks good to go.

File: mkdocs.yml [hunk 1]
```diff
@@ -62,6 +62,9 @@ theme:
 copyright: Copyright &copy; 2024 RACKSPACE TECHNOLOGY
 
 extra:
+  analytics:
+      provider: google
```
Comment (cloudnull) [inline mkdocs.yml:66]:
```suggestion
    provider: google
```

File: mkdocs.yml [hunk 2]
```diff
@@ -62,6 +62,9 @@ theme:
...
+      property: G-NL7YK8QF0X
```
Comment (cloudnull) [inline mkdocs.yml:67]:
```suggestion
    property: G-NL7YK8QF0X
```

**Pull request merged**